src/components/results/sensitivity-analysis/paged-sensitivity-analysis-result.jsx

```diff
@@ -128,16 +128,18 @@ const PagedSensitivityAnalysisResult = ({
     }, [nOrNkIndex, sensiKind, studyUuid, nodeUuid, snackError, intl]);
 
     const fetchResult = useCallback(() => {
-        const { colKey, sortWay } = sortConfig || {};
-
-        const sortSelector =
-            colKey && sortWay
-                ? {
-                      sortKeysWithWeightAndDirection: {
-                          [DATA_KEY_TO_SORT_KEY[colKey]]: sortWay,
-                      },
-                  }
-                : {};
+        const sortSelector = sortConfig?.length
```

Comment:
if sortConfig is an array, I don't think we need to check length, reduce will return default value if it's empty anyway
maybe it was a protection since it's not TS yet ?

Reply:
It's just to avoid adding `sortSelector` with empty `sortKeysWithWeightAndDirection` if the sort array is empty

Reply:
You're right, I did not see the difference at first